CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+cmake_minimum_required(VERSION 3.12)
+
+project(SuperiorMySqlpp VERSION 0.6.1)
+
+option(TEST_ENABLE "Enable test builds" FALSE)
+option(TEST_ODR "Test One Definition Rule" TRUE)
+option(TEST_EXTENDED "Enable extended tests" TRUE)
+
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED TRUE)
+
+configure_file(libsuperiormysqlpp.pc.in
+               ${CMAKE_CURRENT_SOURCE_DIR}/libsuperiormysqlpp.pc @ONLY)
```

Comment:
In-source generation aside, shouldn't the pc file contain the -pthread depedendency?
I'd consider exploring if there's some more systematic way to propagate library's interface link dependencies into the pc file.

More importantly, we're also missing code to export the library target - so we could use following code elsewhere:
```cmake
find_package(SuperiorMySqlpp REQUIRED)
add_executable(Foo main.cc)
target_link_libraries(Foo SuperiorMySqlpp::superiormysqlpp)
```

As we solved both problems elsewhere, please refer to our cmake-utils project.